stories/SQFormDialog.stories.js

```diff
@@ -42,6 +42,7 @@ export const sqFormDialog = () => (
         false,
         'Toggle disabled state of Save Button'
       )}
+      shouldRequireFieldUpdates={true}
```

Comment:
nit: Don't need to specify the `true` value

Reply:
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

React recommends always passing a prop value

Reply:
News to me. Thanks